docs/trex_sandbox_publish.md

```diff
@@ -3,24 +3,18 @@ title: Publish Sandboxed Extensions
 layout: docs
 ---
 
-> *\*\* This is preliminary information and is subject to change \*\**
-
-Sandboxed Extensions are Tableau dashboard extensions that are not permitted to make external network requests. Sandboxed Extensions are hosted by Tableau and run inside an environment that prevents communication with anything except the Tableau hosting server. Sandboxed Extensions are supported in Tableau 2019.4 and later. Available for testing with Tableau 2019.3.
+Sandboxed Extensions are Tableau dashboard extensions that are not permitted to make external network requests. Sandboxed Extensions are hosted by Tableau and run inside an environment that prevents communication with anything except the Tableau hosting server. Sandboxed Extensions are supported in Tableau 2019.4 and later.
 
 ## Before publishing your extension
 
 Be sure to follow the guidelines and requirements to [Create and Test Sandboxed Extensions]({{site.baseurl}}/docs/trex_sandbox_test.html). Make sure your extension works as expected in the development environment. The development environment replicates the Tableau Sandboxed Extension Hosting Cloud Service. If your extension works in the development environment, it will work when you submit the extension package to Tableau for publication.
 
-
 ## Submit your Sandboxed Extension for publication
 
 After you finish developing and testing your Sandboxed Extension, fill out the [Extension Gallery Submission form](https://tabsoft.co/gallerysubmit){:target="_blank"}{:ref="noopener"} with your information and details about your extension.
-For information about what goes in the form, see [Submitting your Extension to the Extension Gallery]({{site.baseurl}}/docs/ux_extension_gallery.html){:target="_blank"}.
-
-Our developers from the Developer Platform team at Tableau will let you know the next steps including legal agreements after you submit your information. If you have any questions about the gallery please send them to [extensiongallery@tableau.com](extensiongallery@tableau.com){:target="_blank"}{:ref="noopener"}.
-
-Once accepted, Tableau will publish your extension and your extension will be available in the [Extensions Gallery](https://extensiongallery.tableau.com/){:target="_blank"}{:ref="noopener"}.  
-
 
+For information about what goes in the form, see [Submitting your Extension to the Tableau Exchange]({{site.baseurl}}/docs/ux_extension_gallery.html){:target="_blank"}.
 
+Our developers from the Developer Platform team at Tableau will let you know the next steps including legal agreements after you submit your information. If you have any questions about the Exchange please send them to [extensiongallery@tableau.com](mailto:extensiongallery@tableau.com){:target="_blank"}{:ref="noopener"}.
```

Comment:
is this the correct email still?

Reply:
Yes this is in my questions doc where I'm confirming with Blake.

Reply:
Confirmed this is correct.